pilot/pkg/networking/plugin/authz/rbac.go

```diff
@@ -463,7 +465,7 @@ func buildTCPFilter(service *serviceMetadata, store model.IstioConfigStore) *lis
 
 	// The result of convertRbacRulesToFilterConfig() is wrapped in a config for http filter, here we
 	// need to extract the generated rules and put in a config for network filter.
-	config := convertRbacRulesToFilterConfig(service, rbacOption{roles: roles, bindings: bindings, forTCPFilter: true})
+	config := convertRbacRulesToFilterConfig(service, rbacOption{roles: roles, bindings: bindings, forTCPFilter: true}, globalPermissive)
```

Comment:
Can we move `globalPermissive` to `rbacOption`?

Reply:
moved.